src/web/releases.rs

```diff
@@ -329,52 +345,64 @@ fn releases_handler(req: &mut Request, release_type: ReleaseType) -> IronResult<
         }
     };
 
-    let releases = {
-        let mut conn = extension!(req, Pool).get()?;
+    let releases = spawn_blocking(move || {
```

Comment:
Is it worth starting to migrate the sql to async below this, using `runtime.block_on(...)` in the sync handlers that share the query?

Reply:
This would mean changing behaviour for the homepage right now, where now the async runtime would be involved. 

I'm not sure if this could be an issue? 

Is there an advantage to doing this instead of `spawn_blocking` here? I remember it's also the recommended way to work with diesel, and also fast enough.

Reply:
I just fully read the comment again (that `postgres` wraps `tokio-postgres`). 

But changing the structure here won't reduce the amount of layering, or am I missing something? 

Or do you mean to actually migrate to `sqlx` or `tokio-postgres`? That's a little more complex due to the sync/async mix & connection pooling.

Reply:
I was thinking of starting the migration to `sqlx` or `tokio-postgres`, probably via just having a separate connection pool. The mix shouldn't be an issue since `spawn_blocking` and `block_on` give symmetric conversions for whichever direction is needed. I guess it's really pretty independent of the handler migrations though.

Reply:
Two pools were also my idea, but I wanted to migrate only the webserver part first, using spawn_blocking and then to sqlx in a later step. 

Mostly to control the amount of change at once